scripts/apps/search/views/multi-image-edit.html

```diff
@@ -29,8 +29,8 @@ <h3 class="modal__heading modal__heading--grow" translate>Upload media</h3>
 
                         <div class="sd-grid-item sd-grid-item--with-click"
                              ng-click="handleItemClick($event, image)"
-                             ng-repeat="image in images track by image._id"
```

Comment:
Why did you replace tracking by `image._id` to `$index`?

Reply:
that slipped through, I was debugging something and made that change, will revert